src/deals/list/deals.scss

```diff
@@ -340,3 +413,32 @@ pbutton button.secondary {
     }
   }
 }
+
+@media screen and (max-width: 400px) {
```

Comment:
I remember we agreed on 
`$MobileWidth: 375px;`

(you can search for "375" and then in the ux/ui channel to see the conversation)

Feel free to bring up suggestions

Reply:
Issue here is for the range from 375px to ~550px, where the 'title' starts to break- then it might look off.
Possible solution is to change the media query to reach max 550px.
In anycase I aggree that it should be either 375px or ~550px and not 400px.